views/pages/ParticipantsPage.jsx

```diff
@@ -1,21 +1,24 @@
 import React from 'react'
-import { Box } from '@mui/material'
+import { Box, Typography, useMediaQuery } from '@mui/material'
 
 import ParticipantsTable from '../tables/ParticipantsTable'
 import ParticipantsSearchForm from '../forms/ParticipantsSearchForm'
 import PageHeader from '../components/PageHeader'
 import useParticipantsList from '../hooks/useParticipantsList'
 
 const ParticipantsPage = () => {
+  const isMobile = useMediaQuery('(max-width:1100px)')
```

Comment:
Should this be handled by a layout component? or is it specific to this page?

Reply:
There is one in the layout component and it uses a different breakpoint than this one. I've renamed the variable 
[8fc6b20](https://github.com/AMP-SCZ/dpdash/pull/561/commits/8fc6b209dd2332cf93ba6881e649bc1bb4a89c3b)

Reply:
Cool, I think the original naming was fine, just wasn't sure if it could be lifted "up a level" to layouts, but sounds like it already is and this is a page specific adjustment 👍